stb_image_write.h

```diff
@@ -230,7 +235,12 @@ static void stbi__stdio_write(void *context, void *data, int size)
 
 static int stbi__start_write_file(stbi__write_context *s, const char *filename)
 {
-   FILE *f = fopen(filename, "wb");
+   FILE *f;
+#ifdef STBI_MSC_SECURE_CRT
+   fopen_s(&f, filename, "wb");
```

Comment:
I'm not sure this code works as is - are you sure `fopen_s()` sets f to NULL if it fails?